core/java/iesi-rest-without-microservices/src/main/resources/application-iesi-metadata.yml

```diff
@@ -1089,26 +1089,19 @@ iesi:
             nullable: false
```

Comment:
The new application-metadata.yml file also needs to be place in the /core/conf folder